OpenShift/helm/nexus-iq/templates/deployment.yaml

```diff
@@ -3,51 +3,55 @@ kind: Deployment
 metadata:
   name: {{ template "iqserver.fullname" . }}
   labels:
-    app: {{ template "iqserver.fullname" . }}
+    {{- include "iqserver.labels" . | nindent 4 }}
```

Comment:
(I'm taking this opportunity to learn and read.) Since these use `nindent`, could we get away with joining these up to their previous line (`labels:`)?   https://banzaicloud.com/blog/creating-helm-charts-part-2/

Reply:
I'm pretty sure this is a stock template except for the bits I obviously hard coded ;-) I've been working with Helm 3